src/tools/plan-generator.ts

```diff
@@ -187,7 +187,7 @@ package ${this.namespace}
 //
 
 ${tryImport('arcs.core.data.*', this.namespace)}
```

Comment:
do we want to specify explicit imports for `arcs.core.data` as well instead of `*`?

Reply:
I think wildcard imports are the pattern we use for codegeneration, for now.

Reply:
Just discovered: After the number of specific imports grows to include all that are needed to the file, my IDE plugin changes the specific imports to a wildcard import.